controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/LogEntry.java

```diff
@@ -69,6 +69,7 @@ public String message() {
                                                     parts[6].replaceAll("\\\\n", "\n")
                                                             .replaceAll("\\\\t", "\t")))
                          .filter(entry -> entry.at().isAfter(from))
+                         .limit(1_000_000)
```

Comment:
1 million lines, that sounds a bit on the high side.

Reply:
Can probably set it lower, too. OOM at 40M lines, so it should be enough to fix _that_ issue, at least.